### PR TITLE
Feature/signup

### PR DESCRIPTION
**Pull request by robertjallen**
## Description ##

Form signup with react-form and styles from figma

## Type of change ##

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Part of a feature.
- [ ] Refactor (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Testing-

## Checklist ##

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] added new dependencies
- [ ] added tests
- [ ] updated documentation
- [x] Looks good on large screens
- [ ] Looks good on mobile

## Organization ##

- [ ] Have you linked this pull request to a [project ticket](https://github.com/Underdog-Devs/website/projects/2)? 

## Review ##

- [ ] Is this a work in progress?
- [x] Is this ready to be reviewed?
- [ ] Have you added two reviewers to this pull request?

### CODE REVIEW

File: components/landing/hero/index.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import Hero from './Hero';
+import Hero from './hero';
```
Comment (JessBonanno) [inline components/landing/hero/index.ts:1]: This should be ./Hero, the files starts with a capital H

File: pages/signup/Signup.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { FaChevronLeft } from 'react-icons/fa';
+import styles from './signup.module.scss';
+
+export default function Signup() {
+	const Back = () => (<button className={styles.back}><FaChevronLeft /> Back </button>);
```
Comment (JessBonanno) [inline pages/signup/Signup.tsx:7]: If you create a variable that returns jsx please have it be its own component, this button could potentially be reused in other places at some point.

File: pages/signup/Signup.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	const { register, handleSubmit, formState: { errors } } = useForm();
```
Comment (JessBonanno) [inline pages/signup/Signup.tsx:8]: Please add typing

File: pages/signup/Signup.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+	const onSubmit = (data) => console.log(data);
```
Comment (JessBonanno) [inline pages/signup/Signup.tsx:9]: typing here as well

File: pages/signup/Signup.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+	console.log(errors);
```
Comment (JessBonanno) [inline pages/signup/Signup.tsx:10]: please clean up console.logs

File: pages/signup/signup.module.scss [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+    }
+	input::placeholder{
+		color: #8692A6;
+		font-family:Source Sans Pro;
+    padding: 2.5%;
+    font-size: 22px;
+	}
+  label{
+    align-self: flex-start;
+    font-size: 22px;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:70]: we have variables for font size, please prefer to use that

File: pages/signup/signup.module.scss [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+    color:#696F79;
+    padding: 2.5% 0;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:72]: we have variables for spacing please prefer to use those

File: pages/signup/signup.module.scss [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+    .submit {
+      outline: none;
+      border: none;
+      background-color: #1565D8;
+      padding: $space-xxs $space-sm;
+      color: #fff;
+      font-family: $linkFont;
+      border-radius: 5px;
+      width: 100%;
+      margin-top: 20vw;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:83]: please use spacing vars for padding and margin

File: pages/signup/signup.module.scss [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+      padding: 2.5%;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:84]: please use spacing vars for padding and margin

File: pages/signup/signup.module.scss [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+	input::placeholder{
+		color: #8692A6;
+		font-family:Source Sans Pro;
+    padding: 2.5%;
+    font-size: 22px;
+	}
+  label{
+    align-self: flex-start;
+    font-size: 22px;
+    color:#696F79;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:71]: please use our text color variables and our fonts

File: pages/signup/signup.module.scss [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+      color: #696F79;
+      border-radius: 6px;
+      align-self: flex-start;
+      width: 100%;
+      padding: 2.5%;
+    }
+	input::placeholder{
+		color: #8692A6;
+		font-family:Source Sans Pro;
+    padding: 2.5%;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:65]: please use spacing vars for padding and margin

File: pages/signup/signup.module.scss [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+    font-size: 22px;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:66]: please use our font size variables

File: pages/signup/signup.module.scss [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+      font-size: 1rem;
+      color: #696F79;
+      border-radius: 6px;
+      align-self: flex-start;
+      width: 100%;
+      padding: 2.5%;
+    }
+	input::placeholder{
+		color: #8692A6;
+		font-family:Source Sans Pro;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:64]: please use our font family variables

File: pages/signup/signup.module.scss [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+      font-family: $primaryTextFont;
+      font-size: 1rem;
+      color: #696F79;
+      border-radius: 6px;
+      align-self: flex-start;
+      width: 100%;
+      padding: 2.5%;
+    }
+	input::placeholder{
+		color: #8692A6;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:63]: please use our font color variables

File: pages/signup/signup.module.scss [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+      outline: none;
+      border: 1px solid #8692A6;
+      padding: 0 $space-sm;
+      font-family: $primaryTextFont;
+      font-size: 1rem;
+      color: #696F79;
+      border-radius: 6px;
+      align-self: flex-start;
+      width: 100%;
+      padding: 2.5%;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:60]: please use spacing vars for padding and margin

File: pages/signup/signup.module.scss [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+      color: $involvedBlue;
+      font-size: 1.4rem;
+    }
+    .input {
+      outline: none;
+      border: 1px solid #8692A6;
+      padding: 0 $space-sm;
+      font-family: $primaryTextFont;
+      font-size: 1rem;
+      color: #696F79;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:56]: please use our font color variables

File: pages/signup/signup.module.scss [hunk 17]
```diff
@@ -0,0 +1,98 @@
...
+    .icon {
+      color: $involvedBlue;
+      font-size: 1.4rem;
+    }
+    .input {
+      outline: none;
+      border: 1px solid #8692A6;
+      padding: 0 $space-sm;
+      font-family: $primaryTextFont;
+      font-size: 1rem;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:55]: please use our font size variables

File: pages/signup/signup.module.scss [hunk 18]
```diff
@@ -0,0 +1,98 @@
...
+    margin-top: 10vw;
+    width: 70%;
+
+    .icon {
+      color: $involvedBlue;
+      font-size: 1.4rem;
+    }
+    .input {
+      outline: none;
+      border: 1px solid #8692A6;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:52]: please use our color variables

File: pages/signup/signup.module.scss [hunk 19]
```diff
@@ -0,0 +1,98 @@
...
+    justify-content: space-around;
+    padding: $space-xxs;
+    border-radius: 5px;
+    margin: 0 auto;
+    margin-top: 10vw;
+    width: 70%;
+
+    .icon {
+      color: $involvedBlue;
+      font-size: 1.4rem;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:48]: please use our font size variables

File: pages/signup/signup.module.scss [hunk 20]
```diff
@@ -0,0 +1,98 @@
...
+  .form {
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    padding: $space-xxs;
+    border-radius: 5px;
+    margin: 0 auto;
+    margin-top: 10vw;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:43]: please use spacing vars for padding and margin

File: pages/signup/signup.module.scss [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+  }
+  .information{
+    margin-top: 10vw;
+  }
+  .back{
+    margin-top: 69px;
+    outline: none;
+    background-color: #fff;
+    border: none;
+    color: #8692A6;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:31]: please use our font color variables

File: pages/signup/signup.module.scss [hunk 22]
```diff
@@ -0,0 +1,98 @@
...
+  p {
+    color: #8692A6;
+    font-family: $primaryTextFont;
+    font-size: 22px;
+  }
+  .information{
+    margin-top: 10vw;
+  }
+  .back{
+    margin-top: 69px;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:27]: please use spacing vars for padding and margin

File: pages/signup/signup.module.scss [hunk 23]
```diff
@@ -0,0 +1,98 @@
...
+    font-family: $primaryHeaderFont;
+    padding-bottom: $space-sm;
+  }
+  p {
+    color: #8692A6;
+    font-family: $primaryTextFont;
+    font-size: 22px;
+  }
+  .information{
+    margin-top: 10vw;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:24]: please use spacing vars for padding and margin

File: pages/signup/signup.module.scss [hunk 24]
```diff
@@ -0,0 +1,98 @@
...
+  h3 {
+    color: #183B56;
+    font-weight: bold;
+    font-family: $primaryHeaderFont;
+    padding-bottom: $space-sm;
+  }
+  p {
+    color: #8692A6;
+    font-family: $primaryTextFont;
+    font-size: 22px;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:21]: please use our font size variables

File: pages/signup/signup.module.scss [hunk 25]
```diff
@@ -0,0 +1,98 @@
...
+	margin-bottom: $space-xl;
+
+  h3 {
+    color: #183B56;
+    font-weight: bold;
+    font-family: $primaryHeaderFont;
+    padding-bottom: $space-sm;
+  }
+  p {
+    color: #8692A6;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:19]: please use our color variables

File: pages/signup/signup.module.scss [hunk 26]
```diff
@@ -0,0 +1,98 @@
...
+	color: $textBlack;
+	background-color: #fff;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	padding: $space-xl;
+	margin-bottom: $space-xl;
+
+  h3 {
+    color: #183B56;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:13]: please use our font color variables

Comment (JessBonanno) [reply pages/signup/signup.module.scss:13]: h3 -> font-size: $f3

File: pages/signup/signup.module.scss [hunk 27]
```diff
@@ -0,0 +1,98 @@
...
+  label{
+    align-self: flex-start;
+    font-size: 22px;
+    color:#696F79;
+    padding: 2.5% 0;
+  }
+    .submit {
+      outline: none;
+      border: none;
+      background-color: #1565D8;
```
Comment (JessBonanno) [inline pages/signup/signup.module.scss:77]: please use our color variables

File: pages/signup/Signup.tsx [hunk 28]
```diff
@@ -0,0 +1,26 @@
...
+
+	return (
+		<div className={styles.container}>
+			<Back />
+			<div className={styles.information}>
+				<h3>Sign Up To Recieve More Information</h3>
+				<p>Someone from the intake team will reach out.</p>
+			</div>
+			<form className={styles.form} onSubmit={handleSubmit(onSubmit)}>
+				<label htmlFor="">email address</label>
```
Comment (JessBonanno) [inline pages/signup/Signup.tsx:20]: please capitalize and fix font

File: pages/signup/Signup.tsx [hunk 29]
```diff
@@ -0,0 +1,26 @@
...
+				<input className={styles.input} type="text" placeholder="enter email address" {...register('Email', { required: true, pattern: /^\S+@\S+$/i })} />
```
Comment (JessBonanno) [inline pages/signup/Signup.tsx:21]: Please capitalize

**Review by JessBonanno — CHANGES_REQUESTED** [review]

File: components/landing/hero/index.ts (hunk 1 above)
Comment (robertjallen) [reply components/landing/hero/index.ts:1]: ✔️

File: pages/signup/Signup.tsx (hunk 2 above)
Comment (robertjallen) [reply pages/signup/Signup.tsx:7]: ✔️

File: pages/signup/Signup.tsx (hunk 3 above)
Comment (robertjallen) [reply pages/signup/Signup.tsx:8]: ?

File: pages/signup/Signup.tsx (hunk 4 above)
Comment (robertjallen) [reply pages/signup/Signup.tsx:9]: ?

File: pages/signup/Signup.tsx (hunk 5 above)
Comment (robertjallen) [reply pages/signup/Signup.tsx:10]: ✔️

File: pages/signup/signup.module.scss (hunk 6 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:70]: ✔️

File: pages/signup/signup.module.scss (hunk 7 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:72]: ✔️

File: pages/signup/signup.module.scss (hunk 8 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:83]: ✔️

File: pages/signup/signup.module.scss (hunk 24 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:21]: ✔️

File: pages/signup/signup.module.scss (hunk 9 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:84]: ✔️

File: pages/signup/signup.module.scss (hunk 10 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:71]: ✔️

File: pages/signup/signup.module.scss (hunk 11 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 12 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 13 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 14 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 15 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 16 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 17 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:49]: ✔️

File: pages/signup/signup.module.scss (hunk 18 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 19 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:42]: ✔️

File: pages/signup/signup.module.scss (hunk 20 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 21 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 22 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 23 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:25]: ✔️

File: pages/signup/signup.module.scss (hunk 25 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:19]: ✔️

File: pages/signup/signup.module.scss (hunk 26 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss]: ✔️

File: pages/signup/signup.module.scss (hunk 27 above)
Comment (robertjallen) [reply pages/signup/signup.module.scss:72]: ✔️

File: pages/signup/Signup.tsx (hunk 28 above)
Comment (robertjallen) [reply pages/signup/Signup.tsx]: ✔️

File: pages/signup/Signup.tsx (hunk 29 above)
Comment (robertjallen) [reply pages/signup/Signup.tsx]: ✔️

**Pull request merged**